File: intranet-angular.Server/Response/MidiaNoticiaResponse.cs
using intranet_angular.Server.Entities;
using intranet_angular.Server.Enuns;

namespace intranet_angular.Server.Response
{
    public class MidiaNoticiaResponse
    {
        public int Id { get; set; }
        public int NoticiaId { get; set; }
        public TipoMidiaEnum Tipo { get; set; }
        public string URL { get; set; }
        public MidiaTamanhoEnum MidiaTamanho { get; set; }

    }
}

File: intranet-angular.client/src/response/midiaNoticiaResponse.ts
import { MidiaTamanhoEnum } from "../app/enum/midia-tamanho.enum";
import { TipoMidiaEnum } from "../app/enum/tipo-midia.enum";

export interface MidiaNoticiaResponse {
  id: number;
  noticiaId: number;
  tipo: TipoMidiaEnum;
  url: string;
  midiaTamanho: MidiaTamanhoEnum;
}
